src/esp/physics/bullet/BulletPhysicsManager.cpp

```diff
@@ -161,6 +161,14 @@ void BulletPhysicsManager::stepPhysics(double dt) {
   int numSubStepsTaken =
       bWorld_->stepSimulation(dt, /*maxSubSteps*/ 10000, fixedTimeStep_);
   worldTime_ += numSubStepsTaken * fixedTimeStep_;
+
+  // Manually sync the motionstates for all DYNAMIC objects. KINEMATIC and
+  // STATIC sync should always be one-way.
+  for (auto& objectItr : existingObjects_) {
+    if (objectItr.second->getMotionType() == MotionType::DYNAMIC) {
```

Comment:
Another idea here: have you considered modifying the dt passed to stepSimulation to be a multiple of fixedTimeStep_, such that Bullet effectively ends up not doing the unwanted interpolation?